src/services/completions.ts

```diff
@@ -2180,6 +2184,18 @@ export function getCompletionEntriesFromSymbols(
     }
 }
 
+function symbolHasValueDeclaration(symbol: Symbol): boolean {
+    return !symbol?.declarations?.length || symbol.declarations.some(declaration => {
+        switch (declaration.kind) {
+            case SyntaxKind.InterfaceDeclaration:
+            case SyntaxKind.TypeAliasDeclaration:
+                return false;
+            default:
+                return true;
+        }

```

Comment:
Is there a better way to check? I searched around for `/is.*type.*only/` & similar but couldn't find anything.

Reply:
what about `symbol.valueDeclaration`?

Reply:
No juice, some declarations are `declare`d without a value declaration. Example from `     Error: New baseline created at tests/baselines/local/tsserver/completions/works-when-files-are-included-from-two-different-drives-of-windows.js`:

```diff
-  {
-    name: "React",
-    kind: "alias",
-    kindModifiers: "",
-    sortText: "11",
-  },
-  {
-    name: "Router",
-    kind: "alias",
-    kindModifiers: "",
-    sortText: "11",
-  },

...

- {
-   "name": "globalThis",
-   "kind": "module",
-   "kindModifiers": "",
-   "sortText": "15"
- },
```

Reply:
Hm, i wonder what’s the exact difference between those and smth like `declare const a: number` - i rechecked this in the ts-ast-viewer before posting and `.valueDeclaration` is set on this.

Either way… i would create a test case with `verbatimModuleSyntax: true`, misuse a symbol and recheck where and why the error is reported in this case - the check that u are looking for is likely used to report that error. And I guess that you could also misuse a symbol without this flag so just any test case with symbol’s misuse should help you out here

Reply:
Commented above; `flags` tells you what meaning(s) a symbol has

Reply:
Aha thank you!